### ISSUE TITLE
Review current status of Python version and dependencies

### ISSUE THREAD

**Issue by AdriiiPRodri** [wazuh/wazuh#9329]
Hi team,

We need to check the status of the current version of Python and all the dependencies installed with Wazuh. For this, we can use tools like [Safety](https://pypi.org/project/safety/).

As for the Python version, we can check websites like [Snyk](https://snyk.io/) or any other tool that fulfills these tasks.

In case of finding some kind of bug or vulnerability, an issue will be created indicating the version of the affected package, the version that patches the bug, and a version suggestion. Example:

### Boto3

```
Current installed version: 1.4.4
Affected version: <1.4.5
Vuln detail: 
Affected versions of this package are vulnerable to Information Exposure due to logging all of the bytes uploaded when the logger is set to the INFO level. 
https://snyk.io/vuln/SNYK-PYTHON-BOTO3-40617

Fixed: >=1.4.5
Suggestion: 1.5.0
```

Regards

**Comment by AdriiiPRodri** [comment]
## Status update

Hi team,

We have reviewed the dependencies that will be included in Wazuh version 4.3, these dependencies are the ones included in the following PR: https://github.com/wazuh/wazuh/pull/9279.

This is the analysis result:

```
root@wazuh-master:/# /var/ossec/framework/python/bin/safety check -r /wazuh/framework/requirements.txt 
+==============================================================================+
|                                                                              |
|                               /$$$$$$            /$$                         |
|                              /$$__  $$          | $$                         |
|           /$$$$$$$  /$$$$$$ | $$  \__//$$$$$$  /$$$$$$   /$$   /$$           |
|          /$$_____/ |____  $$| $$$$   /$$__  $$|_  $$_/  | $$  | $$           |
|         |  $$$$$$   /$$$$$$$| $$_/  | $$$$$$$$  | $$    | $$  | $$           |
|          \____  $$ /$$__  $$| $$    | $$_____/  | $$ /$$| $$  | $$           |
|          /$$$$$$$/|  $$$$$$$| $$    |  $$$$$$$  |  $$$$/|  $$$$$$$           |
|         |_______/  \_______/|__/     \_______/   \___/   \____  $$           |
|                                                          /$$  | $$           |
|                                                         |  $$$$$$/           |
|  by pyup.io                                              \______/            |
|                                                                              |
+==============================================================================+
| REPORT                                                                       |
| checked 71 packages, using free DB (updated once a month)                    |
+==============================================================================+
| No known security vulnerabilities found.                                     |
+==============================================================================+
```

**Comment by AdriiiPRodri** [comment]
## Status update

After reviewing the version of Python currently installed (3.9.2), we have decided that we will upgrade to the latest stable version available, currently version 3.9.6.
